lib/kindle_highlights/client.rb

```diff
@@ -40,6 +42,8 @@ def highlights_for(asin)
     attr_accessor :email_address, :password, :mechanize_options
 
     def conditionally_sign_in_to_amazon
+      retries ||= 0
```

Comment:
What's the motivation to use `||=` instead of just initializing it as `retries = 0`?